src/main/java/org/broadinstitute/consent/http/resources/DacResource.java

```diff
@@ -45,12 +45,7 @@ public DacResource(DacService dacService) {
     public Response findAll(@Auth AuthUser authUser, @QueryParam("withUsers") Optional<Boolean> withUsers) {
         List<Dac> dacs = new ArrayList<>();
         final Boolean includeUsers = withUsers.isPresent() ? withUsers.get() : true;
-        if (includeUsers) {
-            dacs.addAll(dacService.findAllDacsWithMembers());
-        }
-        else {
-            dacs.addAll(dacService.findDacsByUser(authUser));
-        }
+        dacs.addAll(dacService.findDacsByUser(authUser, includeUsers));
```

Comment:
Minor optimization. Get rid of line 46 and just create the list on line 48:
```suggestion
        List<Dac> dacs = dacService.findDacsByUser(authUser, includeUsers);
```